src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -35,6 +34,8 @@ private void initFirstRun() throws SQLException, IOException {
 	private void run(String[] args) throws Exception {
 		this.config = MaxwellConfig.buildConfig("config.properties", args);
 
+		SchemaStore.createMaxwellSchema(this.config.getMasterConnection());
+
```

Comment:
Don't understand this. This needs to happen only exactly once right? So it was at it's right place in first run?